commons/active/questionnaire/questionnaire.avsc

```diff
@@ -6,6 +6,8 @@
   "fields": [
     { "name": "time", "type": "double", "doc": "Timestamp in UTC (s) when the questionnaire is submitted to the subject." },
     { "name": "timeCompleted", "type": "double", "doc": "Timestamp in UTC (s) when subject marks the questionnaire as." },
+    { "name": "timeNotification", "type": "double", "doc": "Timestamp in UTC (s) when the notification to complete the questionnaire is sent." },
+    { "name": "timeZone", "type": "string", "doc": "Local time zone that the device is running in." },
```

Comment:
@blootsvoets do you think this should be UTC offset? Rather than String `timeZone`

Reply:
Yes, indeed, seconds from UTC. In pRMT the variable is called `offset` but that is a topic specifically about the time zone. Also, because these are new variables, we should set a default (`null`) for older values that do not have this variable. Perhaps here it could be:
```suggestion
    { "name": "timeNotification", "type": ["null", "double"], "doc": "Timestamp in UTC (s) when the notification to complete the questionnaire is sent.", "default": null },
    { "name": "timeZoneOffset", "type": ["null", "int"], "doc": "Local time zone offset from UTC (s).", "default": null },
```

Reply:
Yes, indeed, seconds from UTC. In pRMT the variable is called `offset` but that is a topic specifically about the time zone. Also, because these are new variables, we should set a default (`null`) for older values that do not have this variable. Perhaps here it could be:
```suggestion
    { "name": "timeNotification", "type": ["null", "double"], "doc": "Timestamp in UTC (s) when the notification to complete the questionnaire is sent.", "default": null },
    { "name": "timeZoneOffset", "type": ["null", "int"], "doc": "Local time zone offset from UTC (s).", "default": null },
```

Reply:
Yes, indeed, seconds from UTC. In pRMT the variable is called `offset` but that is a topic specifically about the time zone. Also, because these are new variables, we should set a default (`null`) for older values that do not have this variable. Perhaps here it could be:
```suggestion
    { "name": "timeNotification", "type": ["null", "double"], "doc": "Timestamp in UTC (s) when the notification to complete the questionnaire is sent.", "default": null },
    { "name": "timeZoneOffset", "type": ["null", "int"], "doc": "Local time zone offset from UTC (s).", "default": null },
```

Reply:
Yes, indeed, seconds from UTC. In pRMT the variable is called `offset` but that is a topic specifically about the time zone. Also, because these are new variables, we should set a default (`null`) for older values that do not have this variable. Perhaps here it could be:
```
    { "name": "timeNotification", "type": ["null", "double"], "doc": "Timestamp in UTC (s) when the notification to complete the questionnaire is sent.", "default": null },
    { "name": "timeZoneOffset", "type": ["null", "int"], "doc": "Local time zone offset from UTC (s).", "default": null },
```

Reply:
Yes, indeed, seconds from UTC. In pRMT the variable is called `offset` but that is a topic specifically about the time zone. Also, because these are new variables, we should set a default (`null`) for older values that do not have this variable. Perhaps here it could be:
```suggestion
    { "name": "timeNotification", "type": ["null", "double"], "doc": "Timestamp in UTC (s) when the notification to complete the questionnaire is sent.", "default": null },
    { "name": "timeZoneOffset", "type": ["null", "int"], "doc": "Local time zone offset from UTC (s).", "default": null },
```

Reply:
Yes, indeed, seconds from UTC. In pRMT the variable is called `offset` but that is a topic specifically about the time zone. Also, because these are new variables, we should set a default (`null`) for older values that do not have this variable. Perhaps here it could be:
```suggestion
    { "name": "timeNotification", "type": ["null", "double"], "doc": "Timestamp in UTC (s) when the notification to complete the questionnaire is sent.", "default": null },
    { "name": "timeZoneOffset", "type": ["null", "int"], "doc": "Local time zone offset from UTC (s).", "default": null },
```

Reply:
Yes, indeed, seconds from UTC. In pRMT the variable is called `offset` but that is a topic specifically about the time zone. Also, because these are new variables, we should set a default (`null`) for older values that do not have this variable. Perhaps here it could be:
```suggestion
    { "name": "timeNotification", "type": ["null", "double"], "doc": "Timestamp in UTC (s) when the notification to complete the questionnaire is sent.", "default": null },
    { "name": "timeZoneOffset", "type": ["null", "int"], "doc": "Local time zone offset from UTC (s).", "default": null },
```